modules/push-notifications/src/PushNotifications.ts

```diff
@@ -225,35 +226,36 @@ export default class PushNotifications extends ManagedModule<Config> {
   }
 
   private async enableModule() {
```

Comment:
Revert the lifecycle changes, and implement the updateProvider method for the PushNotificationsRoutes and the handler